nc/graphics/Mesh.cpp

```diff
@@ -1,7 +1,11 @@
 #include "debug\NcException.h"
 #include "graphics\d3dresource\GraphicsResourceManager.h"
 #include "graphics\Mesh.h"
-#include "utils\objloader\ObjLoader.h"
+#include <assimp\Importer.hpp>
+#include <assimp\scene.h>
+#include <assimp\postprocess.h>
+#include <vector>
```

Comment:
I've been trying to separate source/language includes like so:

#include "thing.h"
#include "other.h"

#include <vector>
...
What do you think?